src/store/account/sagas.js

```diff
@@ -93,6 +94,7 @@ function* onFetchAccount(account) {
   yield fork(addPlaylistsNotInLibrary)
 
   yield fork(initAudioChecks)
+  yield call(createUserBankIfNeeded)
```

Comment:
Maybe we can put it here instead, because we try to fetch the user bank and attach to the current user obj in the store - so would be nice if it also created if didn't exist here

Reply:
https://github.com/AudiusProject/audius-client/blob/3b414bba7a2c3f6fcfce9af3739a3ddb3e3e52b1/src/services/AudiusBackend.js#L666-L667

Reply:
chatted on slack